src/utils/isDayVisible.js

```diff
@@ -1,7 +1,10 @@
+import moment from 'moment';
+
 import isBeforeDay from './isBeforeDay';
 import isAfterDay from './isAfterDay';
 
 export default function isDayVisible(day, month, numberOfMonths, enableOutsideDays) {
+  if (!moment.isMoment(day)) return false;
```

Comment:
This is a good change, even if my above comments are addressed (either this function should throw or return false on a non-moment object)

Reply:
Followed the same pattern as other utils that do an early return if non-moment object, not sure if it was intended not to have a check but the tests don't break